src/log-manager/src/logrotate/logrotate.conf

```diff
@@ -4,12 +4,11 @@
 /usr/local/pai/logs/*/*/*/*/*.all
 {
     nomail
-    weekly
-    rotate 4
-    compress
+    rotate 1
+    nocompress
     missingok
     notifempty
     copytruncate
-    maxsize 300M
+    size 100M
```

Comment:
So user log can only keep [100M, 200M]? Which is too small.
Let's keep 300M here

Reply:
You mean [300M, 600M]? or [150M, 300M]. I think keep 600MB logs it too large, I'm OK to change to it [150M,300M]

Reply:
How about [256, 512]?

Reply:
256M seems fine. I will raise my concern here

In current config, if log file not exceed `size`. The log file will be keep forever. So if we set size to `256M`, and many jobs scheduled to this node write much log. The max log size will be 256M * number_of_jobs_scheduled _to_this_node (even this job is deleted by FC). That's the reason I don't want to keep too much user log here.

After we make a final log collection solution, this will be solved. But for now, still has such issue.

Reply:
If so why previous 300MB is fine but current 256MB is not?

Reply:
Before, we use `weekly` rotate. Logs exists more than one week will be rotated. And rotated logs exists more than one month will be dropped.
And we compress the rotated logs before.

Reply:
BTW, does the log forever kept? Why not delete the log after its job is deleted in K8s or DB (our next version)? Because after that, user cannot access the job anyway.

Reply:
So, change to rotate by size, we also need to support a way to delete log in this PR

Reply:
...It's not a easy fix and need more efforts. I think this PR is for quick fix and I want to redesign the log collection mechanism in next release.

Reply:
fixed